tidy3d/web/cli/readme.md

```diff
@@ -0,0 +1,17 @@
+## API key
+
+### Generating an API key
+
+You can find your API key in the web http://tidy3d.simulation.cloud
+
+
+### Setup API key
+
+#### Environment Variable
+``export SIMCLOUD_API_KEY="your_api_key"``
+
+#### Command line
+``tidy3d configure``, then enter your API key
```

Comment:
Quick question, I needed to do something like 
```
poetry run tidy3d configure
```
for this to work. How did you set up poetry when testing? Did you use `poetry shell` first to start the virtual env and then run the other commands? `pytest tests` etc?

Reply:
Yes, poetry recommend to use ``poetry shell`` to enter virtual env during developing.